styles.css

```diff
@@ -129,6 +129,120 @@
     padding: 114px 24px;
     background-color: #e5e5e5;
     background-image: none;
+    position: relative;
+  }
+
+  .popup-card {
+    display: none;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    overflow-y: scroll;
+    overflow-x: hidden;
+    backdrop-filter: blur(6px);
+    padding: 40px;
+  }
```

Comment:
- [ ] It is also required to stick to the figma design as much as possible. I would recommend you to make your popup card to be as medium as possible for the desktop so that there wouldn't be any need for scrolling to see the rest of the information on the card. Kindly look at the figma design and work your design alongside that.

### Your Design
![your-window](https://user-images.githubusercontent.com/26597432/224416845-630f4a3f-158b-48db-9f0d-b65c7039fae2.png)

### Figma Design
![temp-window](https://user-images.githubusercontent.com/26597432/224416892-b9b01c62-96bc-4ebe-b11d-8186575e648d.png)

Reply:
I don't think that's possible. The box is only as large as its content. I can't adjust its height